pkg/services/searchV2/testdata/folders-dashboard-removed-on-folder-removed.jsonc

```diff
@@ -27,7 +34,14 @@
         "meta": {
           "type": "search-results",
           "custom": {
-            "count": 1
+            "count": 1,
+            "locationInfo": {
+              "general": {
+                "name": "General",
+                "kind": "folder",
+                "url": "/dashboards/f/general/"
```

Comment:
The general/root folder is just:
https://play.grafana.org/dashboards

Only folders with a UID live under:
https://play.grafana.org/dashboards/f/tOFItmonk/grafanacloud

Reply:
Yeah, I know that, but we still use UID `general` to operate with the general folder abstraction in the backend. As I commented [here](https://github.com/grafana/grafana/pull/60729#discussion_r1082388334) both access control and search v2 itself use it.